src/modules/edc-demo/components/transfer-history/transfer-history-viewer.component.ts

```diff
@@ -1,16 +1,20 @@
 import {Component, OnInit} from '@angular/core';
 import {MatDialog} from '@angular/material/dialog';
-import {Observable, of} from 'rxjs';
 import {map} from 'rxjs/operators';
 import {
   TransferProcessDto,
   TransferProcessService,
 } from '../../../edc-dmgmt-client';
+import {Fetched} from '../../models/fetched';
 import {
   ConfirmDialogModel,
   ConfirmationDialogComponent,
 } from '../confirmation-dialog/confirmation-dialog.component';
 
+export interface TransferProcessesList {
+  transferProcesses: TransferProcessDto[];
+}
```

Comment:
no, this was not what was meant. There is no need for a "TransferProcessList" interface/type. "TransferProcessDto[]" is the transfer process list.